pop-cli-for-appchains/SUMMARY.md

```diff
@@ -14,6 +14,7 @@
   * [Create a Contracts parachain](guides/create-a-new-parachain/create-a-smart-contract-parachain.md)
   * [Create an EVM parachain](guides/create-a-new-parachain/create-an-evm-parachain.md)
 * [Build your parachain](guides/build-your-parachain.md)
+* [Calling a chain](guides/calling-a-chain.md)
```

Comment:
```suggestion
* [Call a chain](guides/call-a-chain.md)
```

Reply:
I wanted to keep consistency with the rest. The contracts guide is 
`* [Calling your contract](guides/calling-your-contract.md)`

Reply:
But you are right, seems better Call. Fixed here https://github.com/r0gue-io/pop-docs/pull/12/commits/e1d222a151a10fd569f533cdbe9a0a5ebefc18ae, I will solve in the call contract too.